src/signal.rs

```diff
@@ -1706,34 +1707,111 @@ mod editable {
     }
 }
 
+pub trait SpinButtonSignals {
```

Comment:
It in this file, only one without `xxSignals`.
Not sure, that it really need be fixed. IMHO better wait for gkoz.